sessions/recap-project-2/quiz-app/README.md

```diff
@@ -0,0 +1,90 @@
+# Quiz App
```

Comment:
recap project 2 does not have a package json so i'll use this random line to comment…

When ghcd does not find a package json it tries to mash together a folder name and it will become something like:

web-exercises-quiz-app

We can of course change that behaviour to just use the dirname.

We could also add a simple package.json that does nothing besides having a name.

Or implement the addition to ghcd described above.

Reply:
Oh, good point! 

I added a `package.json` file. 

From the first HTML basic exercises on there is a `package.json`file. Students should be used to it, even if they don't know the purpose yet.